kernel/libc/newlib/newlib_isatty.c

```diff
@@ -20,6 +20,7 @@ int isatty(int fd) {
 
 int _isatty_r(struct _reent *reent, int fd) {
     (void)reent;
+    (void)fd;
```

Comment:
Why is this needed if fd is used in the function?

Reply:
Good point. I had modified the function to take out the following check temporarily, added that to suppress the warning, then put the check back in and forgot to remove this. lol.